src/NSwag.AspNetCore.Launcher/NSwag.AspNetCore.Launcher.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp1.0;net461</TargetFrameworks>
+    <TargetFrameworks>netcoreapp1.0;net461;netstandard1.6</TargetFrameworks>
```

Comment:
Not super sure it helps - it's looking for it in a different package's directory `/Users/charles/.nuget/packages/nswag.commands/11.17.9/lib/netstandard1.6`, not just a different TFM for the console core package